<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Record</class>
 <widget class="QWidget" name="Record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>文泉驿等宽微米黑</family>
    <pointsize>15</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color: rgb(255, 255,255);
	font: 15pt &quot;文泉驿等宽微米黑&quot;;
	background-color: rgb(0, 50, 105);
}
QPushButton{
	border: 1px solid ; 
	border-color: rgb(53, 156, 212);
	border-radius:10px;
}
QComboBox {
border:1px solid rgb(53, 156, 212);
}
QLable{
color: rgb(255, 255,255);
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox_recode">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>790</width>
     <height>470</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
font: 14pt &quot;文泉驿等宽微米黑&quot;;
border: 1px solid white;
border-radius:8px;
margin-top:10px;
}
QGroupBox:title{
color:white;
subcontrol-origin: margin;
left: 5px;
}</string>
   </property>
   <property name="title">
    <string>记录查询</string>
   </property>
   <widget class="QTableWidget" name="tableWdt_record">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>85</y>
      <width>780</width>
      <height>330</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿等宽微米黑</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
font: 13pt &quot;文泉驿等宽微米黑&quot;;
gridline-color: rgb(255, 255, 255);
}
QTableWidgetItem{
font: 13pt &quot;文泉驿等宽微米黑&quot;;
}
</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>780</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
spacing: 5px;
color: rgb(0, 0, 0);
border:1px solid;
border-radius:10px;
background-color: rgb(217, 70, 38);
}
QCheckBox::indicator{
width:12px;
height:12px;
margin-left:5px;
image: url(:/Image/uncheck.png);
}

QCheckBox::indicator::unchecked {
image: url(:/Image/uncheck.png);
}
QCheckBox::indicator::checked {
image: url(:/Image/checked.png);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lb_stat">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>67</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>起始时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_end">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>45</y>
       <width>67</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>结束时间：</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dTEdit_end">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>45</y>
       <width>190</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid  rgb(58, 152, 234);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd hh:mm:ss</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dTEdit_start">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>190</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid  rgb(58, 152, 234);
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd hh:mm:ss</string>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="QCom_del">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>15</y>
       <width>92</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>当前一条</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>当前一页</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>所有记录</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lb_delType">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>15</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>记录选择:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_dataType">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>45</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>数据类型:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cbBox_type">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>45</y>
       <width>92</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxCount">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>全部记录</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>节点报警</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>模块故障</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通讯故障</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_net">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>17</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>通道</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pass">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>15</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_id">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>47</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_address">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>45</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_del">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>25</y>
       <width>85</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
     <property name="icon">
      <iconset resource="../../Bevone-PowerMonitor/image.qrc">
       <normaloff>:/Image/recycle.png</normaloff>:/Image/recycle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_check">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>25</y>
       <width>85</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/search.png</normaloff>:/Image/search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <zorder>lb_stat</zorder>
    <zorder>lb_end</zorder>
    <zorder>dTEdit_end</zorder>
    <zorder>dTEdit_start</zorder>
    <zorder>lb_delType</zorder>
    <zorder>lb_dataType</zorder>
    <zorder>QCom_del</zorder>
    <zorder>cbBox_type</zorder>
    <zorder>checkBox_net</zorder>
    <zorder>lineEdit_pass</zorder>
    <zorder>checkBox_id</zorder>
    <zorder>lineEdit_address</zorder>
    <zorder>pBtn_del</zorder>
    <zorder>pBtn_check</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>410</y>
      <width>781</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lb_current">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pageNum">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>46</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(10, 10, 10);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>页数</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>20</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_jump">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>18</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>跳转到</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_upPage">
     <property name="geometry">
      <rect>
       <x>505</x>
       <y>18</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>上一页</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/fore.png</normaloff>:/Image/fore.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_downPage">
     <property name="geometry">
      <rect>
       <x>595</x>
       <y>18</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>下一页</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/back.png</normaloff>:/Image/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_quit">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>18</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid ;
border-color: rgb(226, 79, 79);
border-radius:13px;</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/go-back.png</normaloff>:/Image/go-back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pBtn_pagePrint">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>18</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>打印</string>
     </property>
     <property name="icon">
      <iconset resource="../../Bevone-PowerMonitor/image.qrc">
       <normaloff>:/Image/print.png</normaloff>:/Image/print.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count_2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>共</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>条</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>21</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>页</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_countNum">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>20</y>
       <width>45</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>页</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>第</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>页</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_count_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>共</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../Bevone-PowerMonitor/image.qrc"/>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
